### DISCUSSION TITLE
ERC 1155 / ERC 721 fetching metadata issue

### DISCUSSION THREAD

**Discussion by ssunicsoft** [blockscout/blockscout#7005] [Q&A]
If there special ENV to enable fetching metadata for NFT tokens?
I deployed blockscout for my local EVM network (based on EVMOS).
It indexes ERC 1155 / 721 tokens, but the token page has HTTP 404. 
If anyone meets such behaviour? I use blockscout 5.1.0
Plz help...
http://ec2-3-234-221-169.compute-1.amazonaws.com:4000/address/0x88865eA80998B00cF2BBf0303E96cD91495A4873/tokens#address-tabs

![0x88865eA80998B00cF2BBf0303E96cD91495A4873 - Frontier chain Explorer - Google Chrome 2023-03-01 15 00 52](https://user-images.githubusercontent.com/94377662/222121385-1fc7ce45-9004-4a73-a789-4c5eb4562a2c.png)
![Frontier chain Explorer - BlockScout - Google Chrome 2023-03-01 15 01 57](https://user-images.githubusercontent.com/94377662/222121422-0f7c0f8a-a4ea-40a4-a6b8-196d37a3b916.png)

**Comment by Jackalgirl** [comment]
I'm curious about this as well; I'm having the same issue while deploying an ERC-721 contract on the new Lamina1 testnet.  The weird thing is that it's intermittent.  Some tokens I mint work just fine, in the sense that you can navigate to the individual token page and it shows up without a problem.  Others, though, throw a 404 error -- all with the exact same contract.

I haven't been able to suss out a pattern; I've tried "safeminting" NFTs via both Remix and the Write Contract option in Blockscout and there's no clear situation that will cause a token to work (or not).

Originally, I'd thought that the issue was with the metadata, but I've gone over the files a thousand times, and different tokens have been affected from attempt to attempt.  The next thought was that the Solidity version / compiler was the problem, so I made sure that those were the same, but while that moved me from "no tokens visible" to "some tokens visible", it didn't result in a consistent experience.

I'm now wondering if this has to do with the way that Blockscout works.  Is it possible that, during the execution of a contract, that Blockscout somehow "misses" an entry, or drops some data?  Is there a way to go back and fix the information?  Or what else might be the issue here?  (I half feel like there is some glaringly obvious thing I'm just missing.)

Here are some examples of what I'm talking about:

A token that shows up just fine: [https://testnet-explorer.lamina1.global/token/0xc4feaad379c386a6345e832769e6cb5b77c1d36c/instance/3/token-transfers](https://testnet-explorer.lamina1.global/token/0xc4feaad379c386a6345e832769e6cb5b77c1d36c/instance/3/token-transfers)
A token that doesn't: [https://testnet-explorer.lamina1.global/token/0xc4feaad379c386a6345e832769e6cb5b77c1d36c/instance/4/token-transfers](https://testnet-explorer.lamina1.global/token/0xc4feaad379c386a6345e832769e6cb5b77c1d36c/instance/4/token-transfers)
Tx for the minting of the token that works: [https://testnet-explorer.lamina1.global/tx/0x0fbd09d6da63ea6c973df087a40fcc615d1f022b9a7317c679e44867048916a0](https://testnet-explorer.lamina1.global/tx/0x0fbd09d6da63ea6c973df087a40fcc615d1f022b9a7317c679e44867048916a0)
Tx for the minting of the token that doesn't: [https://testnet-explorer.lamina1.global/tx/0x92cf4dce9052478eec69adfd287863e6a4ad77fc8619d423f669c5ba5857352e](https://testnet-explorer.lamina1.global/tx/0x92cf4dce9052478eec69adfd287863e6a4ad77fc8619d423f669c5ba5857352e)
The Token Inventory (you can see all of the NFT art for tokens 0 through 3, but not 4): [https://testnet-explorer.lamina1.global/token/0xc4FEaAD379C386a6345E832769E6CB5b77C1d36C/inventory](https://testnet-explorer.lamina1.global/token/0xc4FEaAD379C386a6345E832769E6CB5b77C1d36C/inventory)
Token #4 looking fairly basic, but clearly functional, over at nft.storage: [https://bafybeihpuv73hm73bw4fpkoablw7j4lc3qb22mmfllltxm2anjyjm4sxfe.ipfs.nftstorage.link/4.png](https://bafybeihpuv73hm73bw4fpkoablw7j4lc3qb22mmfllltxm2anjyjm4sxfe.ipfs.nftstorage.link/4.png)

Thank you in advance!

**Reply by ssunicsoft** [reply]
@Jackalgirl Hi,
Thank you for your responce.

What version do you use?
And if it is not a secret, can you show envs params?

**Comment by Mojmir-R** [comment]
Hey @Jackalgirl @ssunicsoft, I personally can't directly advise on the issue but you might try searching in/asking in Blockscout's official Discord channel: discord.gg/blockscout Someone from the community or core team might be able to help.

**Reply by Jackalgirl** [reply]
Hi, Mojmir, 
Thank you, I'll do that! : )

**Reply by ssunicsoft** [reply]
Hi, @Mojmir-R 
Thank you for advice :-)
